src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/controllers/KalendarController.java

```diff
@@ -43,27 +43,24 @@
     AuthorizeKal authorizeKal;
 
     @GetMapping(value = "/calendar")
-    public ResponseEntity getKalendarList(@RequestHeader("X-Client-Token") String clientToken, HttpServletRequest request) throws IOException {
-        if (!request.getHeader("X-Client-Token").equals("")) {
-            KalendarListResponse kalendarListResponse = new KalendarListResponse();
-            KalUser user = kalUserRepository.findByClientToken(clientToken);
-            List<Kalendar> list = kalendarService.findKalendars(user);
-            kalendarListResponse.setKalendars(kalendarService.setKalendarResponse(list));
+    public ResponseEntity getKalendarList(@RequestHeader("X-Client-Token") String clientToken) {
+        try {
+            KalendarListResponse kalendarListResponse = kalendarService.makeKalendarListResponse(clientToken);
```

Comment:
I was thinking about renaming this method to be more descriptive - so when you check the KalendarService class it is meaningful and you do not need to check this calles method to understand what is happening.
Maybe `kalendarService.getKalendarsByUser` or `kalendarService.getKalendarsByClientToken` ?